docs/sources/configuration/_index.md

```diff
@@ -3541,6 +3541,11 @@ redis:
   # CLI flag: -<prefix>.redis.max-connection-age
   [max_connection_age: <duration> | default = 0s]
 
+  # Cache config for index entry writing.Enable routing read-only commands to
```

Comment:
I think we should add a link here, or more details, which explains what this option does and why someone should use it.

Reply:
Sure thing, @dannykopping! It's just that I can't find `go-redis` v8's documentation that Loki is currently using (only v6 & v9 on https://pkg.go.dev/github.com/redis/go-redis), so I added more details, please check if it's appropriate.